_episodes/03-inventory.md

```diff
@@ -45,14 +45,14 @@ Checking in equipment:
 
 ### Changing status of equipment
 
-Sometimes, you need to change the status of equipment in order to checkout the asset. The status will also need to be changed for units which are being shipped for RMA.
+Sometimes, you need to change the status of equipment in order to checkout the asset. The status will also need to be changed for units which are being shipped for RMA, or are lost.
```

Comment:
no comma before "or are lost"

Reply:
in my head, switching the order to "units which are lost or being shipped for RMA." sounds better?